bin/GameIndex.dbf

```diff
@@ -1706,6 +1728,21 @@ Serial = SCES-51463
 Name   = Ghosthunter
 Region = PAL-M6
 Compat = 5
+[patches = 3bd85da4]
+	author=Prafull
+	// Fix slow and choppy gameplay
+	//PCSX2 attempts to run VU0 for thousands of cycles as soon as it is started,
```

Comment:
Comments can be better formatted to match the rest.

Reply:
How did i miss this? I was sure i formatted correctly. Thanks